benches/Cargo.toml

```diff
@@ -10,9 +10,9 @@ async-channel = "1.4.0"
 v1={path="../protocols/v1", package="sv1_api"}
 serde_json = { version = "1.0.64", default-features = false, features = ["alloc"] }
 iai="0.1"
-mining_sv2 = { path = "../protocols/v2/subprotocols/mining", version = "^0.1.0" }
+mining_sv2 = { path = "../protocols/v2/subprotocols/mining", version = "^1.0.0" }
 roles_logic_sv2 = { path = "../protocols/v2/roles-logic-sv2" }
```

Comment:
why there's not a version specified here?

Reply:
basically it was already missing, but I'm adding it here https://github.com/stratum-mining/stratum/pull/785/commits/a877dd2960832b743fdbc0bd041f89b10fb252a8